app/src/main/res/layout/activity_parking_stay.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background_parking_stay"
+    tools:context=".activity.ParkingStayActivity">
+
+    <TextView
+        android:id="@+id/start_date_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/date_start_text"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
+        android:textColor="#FFFEFE"
```

Comment:
Look in the other TextVIew, you are using the same properties , so you could use a Style to share these properties

Reply:
done